### PR TITLE
[3.8] bpo-10544: Disallow "yield" in comprehensions and generator expressions.

### PR DESCRIPTION
**Pull request by serhiy-storchaka**
https://bugs.python.org/issue10544

### CODE REVIEW

**Review by ncoghlan — CHANGES_REQUESTED** [review]
Just adding a blocking review to ensure we don't merge this until after the 3.7 maintenance branch has been created.

**Comment by serhiy-storchaka** [comment]
Could you please look on this again Nick?

File: Misc/NEWS.d/next/Core and Builtins/2017-11-26-00-59-22.bpo-10544.fHOM3V.rst
Comment (ncoghlan) [inline Misc/NEWS.d/next/Core and Builtins/2017-11-26-00-59-22.bpo-10544.fHOM3V.rst]: Minor wording tweak: now are -> are now

**Review by ncoghlan — APPROVED** [review]
LGTM, and the main branch is now open for 3.8 commits.

**Comment by serhiy-storchaka** [comment]
Maybe you can suggest better wording for What's New?

**Comment by ncoghlan** [comment]
The What's New wording seemed fine to me.

**Comment by serhiy-storchaka** [comment]
Thank you @ncoghlan for your review.

**Pull request merged**
